api/src/main/java/pl/kolendateam/dadcard/abilitys/entity/AbilityBonus.java

```diff
@@ -0,0 +1,28 @@
+package pl.kolendateam.dadcard.abilitys.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.Setter;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@Entity
+public class AbilityBonus {
+
```

Comment:
Czemu trzymamy to w osobnej tabeli?

Reply:
bo wartości siły, zręczności, są nadawane przez gracza a potem z nich trzeba policzyć bonus w osobnym procesie